server/server.js

```diff
@@ -56,35 +60,38 @@ exports = {
     }
     issue_priority = priorityMap[`${priority_no}`];
 
-   find(args.data.ticket.id).then( data => {
-      const issueID = data.issue_data.issueID;
-      $request.put(`https://api.bitbucket.org/2.0/repositories/${args.iparams.bitBucket_repo}/issues/${issueID}`,{
-        headers: {
-          Authorization : "Bearer <%= access_token %>"
-        },
-        isOAuth: true,
-        json: {
-          "priority" : issue_priority
-        }
+    try {
+        const data = await find(args.data.ticket.id)
+        if(!data) throw new Error(`Error: couldn't able to find issue id, ${data}`);
+        const issueID = data.issue_data.issueID;
+        const res = await $request.put(`https://api.bitbucket.org/2.0/repositories/${args.iparams.bitBucket_repo}/issues/${issueID}`,{
+          headers: {
+            Authorization : "Bearer <%= access_token %>"
+          },
+          isOAuth: true,
+          json: {
+            "priority" : issue_priority
+          }
+
+        })
+        if(res.status !== 200) throw new Error(`Error: Failed to update issue priority,${res.status}`);
 
-      }).then(() => {
         console.log("Successfully changed the issue priority");
```

Comment:
Single quotes for strings over double. 

```suggestion
        console.log('Successfully changed the issue priority');
```